docs/shared/reference/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+Major Nx versions are released as the _latest_ every six months, typically around April and October.
+After each major version release, the _previous_ major version moves to long-term support (LTS) for 12 months, after
+which it becomes unsupported.
+Each major version has 18 total months of support from when it is first released to when it falls out of LTS.
+Workspaces should only use major versions that are under LTS.
+
+## Supported Versions
+
+The following are the currently supported versions of Nx.
+
+| Version | Release Date |
+| ------- | ------------ |
+| v19     | 2024-05-06   |
+| v18\*   | 2024-02-03   |
+| v17     | 2023-10-19   |
+| v16     | 2023-04-27   |
+
+**\*Note:** v18 is a special release and does not fit into the normal release cycle. Thus, v16 continues to be supported
+according to schedule.
+
+### LTS Fixes
+
+LTS fixes are generally considered if they address a security vulnerability or a critical bug that impedes the use of
```

Comment:
This is worded weirdly as well. Maybe something more along the lines of...

Current vs LTS

The current version of Nx will receive new features as well as any fixes to any unintentional behavior. When a new major version of Nx is released, previous versions will go into Long Term Support (LTS). LTS versions of Nx will receive security patches as well as critical fixes.